kernel/build-kernel.sh

```diff
@@ -202,7 +202,10 @@ build_kernel() {
 	arch_target=$(arch_to_kernel "${arch_target}")
 	pushd "${kernel_path}" >>/dev/null
 	make -j $(nproc) ARCH="${arch_target}"
+	if [ "$arch_target" != "powerpc" ]
+	then
```

Comment:
Please could you either indent code blocks (here and below) inside `if` tests.

However, in this case you could simplify to:

```
[ "$arch_target" != "powerpc" ] && [ -e "arch/${arch_target}/boot/bzImage" ] || [ -e "arch/${arch_target}/boot/Image.gz" ]
```